lib/EnvoyAPI.js

```diff
@@ -45,7 +47,13 @@ class EnvoyAPI {
       throw new Error('No token supplied.');
     }
 
+    this.token = token;
     this.baseUrl = process.env.ENVOY_BASE_URL || 'https://app.envoy.com';
+    this.newUrl = 'https://api.envoy.com/v1';
```

Comment:
We should probably call this something more descriptive than `newUrl`, maybe `baseApiUrl` and add an env var for it. Also, we should probably remove the `v1` on it so that it's just the api.envoy.com domain.